### PR TITLE
chore(build): Remove circular dependency in Angular 2 ES5 output.

### PR DESCRIPTION
**Pull request by vikerman**
Remove couple of circular dependency between modules in Angular 2 ES5 output caused by exception_handler.ts and router_providers.ts.

Fix the build/checkCircularDependency gulp task to call madge properly to detect the circular deps.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -344,9 +344,9 @@ gulp.task('lint', ['build.tools'], function() {
 gulp.task('build/checkCircularDependencies', function(done) {
   var madge = require('madge');
 
-  var dependencyObject = madge(CONFIG.dest.js.dev.es5, {
+  var dependencyObject = madge([CONFIG.dest.js.dev.es5], {
     format: 'cjs',
-    paths: [CONFIG.dest.js.dev.es5],
+    paths: undefined,
```
Comment (vsavkin) [inline gulpfile.js]: you don't have to paths at all

Comment (vikerman) [inline gulpfile.js]: Done.

**Comment by vikerman** [comment]
Hi

**Comment by vikerman** [comment]
Hi - https://test.corp.google.com/ui#cl=116341878

**Pull request closed without merging**